bundles/org.openhab.binding.luftdateninfo/src/main/java/org/openhab/binding/luftdateninfo/internal/dto/SensorData.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.luftdateninfo.internal.dto;
+
+import java.util.List;
+
+/**
+ * The {@link SensorData} Data Transfer Object
+ *
+ * @author Bernd Weymann - Initial contribution
+ */
+public class SensorData {
+    private Long id;
+    private String timestamp;
+    private Integer sampling_rate;
+    private List<SensorDataValue> sensordatavalues;
```

Comment:
```suggestion
    private List<SensorDataValue> sensorDataValues;
```

Reply:
Same here - field is exactly named after Json key

Reply:
Solved with SerializedName annotation